src/pybind/mgr/dashboard/frontend/src/app/shared/validators/validators.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  AbstractControl,
+  ValidationErrors,
+  ValidatorFn,
+  Validators as AngularValidators
+} from '@angular/forms';
+
+export type Prerequisites = { // tslint:disable-line
```

Comment:
No, it's not necessary to export the Prerequisites type, this has been fixed now. But i would like to export the ``isEmptyInputValue`` function because it is really helpful if you implement local validators in a form component. This is done in the [RGW user form](https://github.com/votdev/ceph/blob/dashboard_rgw_management/src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-user-form/rgw-user-form.component.ts#L253) for example.